ValveResourceFormat/ValveResourceFormat.csproj

```diff
@@ -1,116 +1,13 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{E685E563-5F49-4D8D-8342-A7354229E54F}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>ValveResourceFormat</RootNamespace>
-    <AssemblyName>ValveResourceFormat</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Why not consume the .NET Standard library under .NET Framework? Are you trying to avoid all of the glue assemblies?

Reply:
It won't build GUI without it due to `System.Drawing.Common`.